src/lucky/force_ssl_handler.cr

```diff
@@ -12,25 +12,24 @@
 
 # *Enabled* - The handler can be enabled/disabled. This is helpful for working
 # in a local development environment.
-
-# *HSTS* - Settings to configure HSTS header with max-age and includeSubDomains
+#
+# *Strict-Transport-Security* - Settings to configure the ['Strict-Transport-Security' header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security)
 #
 # ```
 # # Usually in config/force_ssl_handler.cr
 # Lucky::ForceSSLHandler.configure do |settings|
 #   settings.redirect_status = 303
 #   settings.enabled = false
-#   settings.hsts = {max_age: 18.weeks, include_subdomains: false}
+#   settings.strict_transport_security = {max_age: 1.year, include_subdomains: true}
 # end
 # ```
 class Lucky::ForceSSLHandler
   include HTTP::Handler
-  alias Hsts = NamedTuple(max_age: Time::Span, include_subdomains: Bool)
```

Comment:
The alias was nice, but it made it hard to tell what the setting expected when configuring it. It just says `Hsts`. I was hoping Crystal would also expand the alias and tell you what the NamedTuple was but it doesn't :(